btsieve/Cargo.toml

```diff
@@ -26,6 +26,7 @@ hyper = "0.12"
 itertools = "0.8"
 log = "0.4"
 pretty_env_logger = "0.3"
+rand = "0.7"
```

Comment:
Don't forget to remove this again once we go for deterministic id generation

Reply:
Added to #1249.